context/app/static/js/components/workspaces/WorkspacesList.jsx

```diff
@@ -67,7 +67,8 @@ function WorkspacesList() {
       />
       <Paper>
         {workspacesList.map((workspace) => (
-          <div key={workspace.id}>
+          /* Inbound links have fragments like "#workspace-123" */
+          <div key={workspace.id} id={`workspace-${workspace.id}`}>
```

Comment:
You'll need to offset for the sticky header when using an anchor link as we have for the `Section` component in `js/shared-styles/sections/Section`.

Maybe use `Section` or create a reusable component with shared logic that is more fitting.

Reply:
Thanks for reminder! Filed issue for follow-up.